inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp

```diff
@@ -176,6 +175,26 @@ void MKLDNNPadNode::createPrimitive() {
 
     params.sizeData = this->getSelectedPrimitiveDescriptor()->getConfig().inConfs[0].desc->getPrecision().size();
 
+    if (inputShapesDefined()) {
+        prepareParams();
+        updateLastInputDims();
+    }
+}
+
+void MKLDNNPadNode::prepareParams() {
+    const auto& plnSrcDims = getParentEdgeAt(DATA_ID)->getMemory().getStaticDims();
+    if (padMode == REFLECT) {
+        for (size_t i = 0; i < plnSrcDims.size(); i++) {
+            if ((plnSrcDims[i] - 1) < padsBegin[i] || (plnSrcDims[i] - 1) < padsEnd[i])
+                THROW_ERROR <<  "has incorrect padsBegin or padsEnd for 'reflect' pad mode";
+        }
+    } else if (padMode == SYMMETRIC) {
+        for (size_t i = 0; i < plnSrcDims.size(); i++) {
+            if (plnSrcDims[i] < padsBegin[i] || plnSrcDims[i] < padsEnd[i])
+                THROW_ERROR <<  "has incorrect padsBegin or padsEnd for 'symmetric' pad mode";
+        }
+    }
```

Comment:
please see comments https://github.com/openvinotoolkit/openvino/pull/8375#discussion_r742019188

Reply:
Right, done, thank you